README.md

```diff
@@ -121,3 +122,30 @@ may have the following optional properties in addition to the standard
   `this.inputPaths`. (The name `treeDir` is for historical reasons.)
 - `line`: Line in which the error occurred (one-indexed)
 - `column`: Column in which the error occurred (zero-indexed)
+
+### fsFacade
+
+```js
+const Plugin = require('broccoli-plugin');
+
+class MyPlugin extends Plugin {
+  constructor(inputNodes, options = {}) {
+    super(inputNodes, {
+      annotation: options.annotation,
+      fsFacade: true,
+    });
+  }
+
+  build() {
+    // Read files from this.inputPaths, and write files to this.outputPath.
+    // Silly example:
+
+    // Read 'foo.txt' from the third input node
+    const input = this.input.fs.readFileSync(`foo.txt`);
+    const output = someCompiler(input);
+
+    // Write to 'bar.txt' in this node's output
+    this.output.fs.writeFileSync(`bar.txt`, output);
```

Comment:
This should be `this.output.writeFileSync`. we should also call out that not all methods are allowed in the comment and point to the list that we do.